### PR TITLE
add fire and forget option to python sdk service connector requests

### LINKED ISSUE #41: add "fire and forget" option to python sdk service connector protocol
This addition helps accommodate long running requests with the potential to exceed any service connector request timeout setting. 

If necessary, It is up to the client and connector to work out a mechanism for calling back once the request completes

### PR DESCRIPTION
**Pull request by mer2github**
Closes #41

### CODE REVIEW

File: pythonsdk/build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ ext {
         pypi_repo = 'testpypi'
     }
     if (!project.rootProject.hasProperty('vantiqServiceSdkVersion')) {
-        vantiqServiceSdkVersion = '0.1.0'
+        vantiqServiceSdkVersion = '0.2.0'
```
Comment (mer2github) [inline pythonsdk/build.gradle:16]: want to bump the release number. assuming minor release (ala semantic versioning) is the right number to move.

File: pythonsdk/src/main/python/vantiqservicesdk.py [hunk 2]
```diff
@@ -148,14 +149,33 @@ def __complete_request(completed_task):
...
-                    # Add the task to the set of active requests and remove when done.
-                    # See https://docs.python.org/3/library/asyncio-task.html#creating-tasks
-                    active_requests.add(task)
-                    self._active_requests.inc(1)
-                    task.add_done_callback(__complete_request)
+                    # Decode the message as JSON, but make sure to catch any errors
+                    try:
+                        request = json.loads(msg_bytes.decode("utf-8"))
+                    except Exception as e:
+                        # couldn't deserialize the request, send back an error, but there's no way to know the requestId
```
Comment (mer2github) [inline pythonsdk/src/main/python/vantiqservicesdk.py:156]: when deser fails, we don't get any requestId to use with the error response. not sure how useful the error response is in this situation

**Review by sfitts — APPROVED** [review]
LGTM

**Comment by sfitts** [comment]
I will deploy this tomorrow...

**Pull request merged**